samples/bluetooth/mesh/light_switch/README.rst

```diff
@@ -92,6 +107,62 @@ If the model is configured to publish to a unicast address, the model handler ca
 The response from the target device updates the corresponding LED on the mesh light switch device.
 If the model is configured to publish to a group address, it calls :c:func:`bt_mesh_onoff_cli_set_unack` instead, to avoid getting responses from multiple devices at once.
 
+Low Power node support
+======================
+
+The mesh light switch sample can also be run as a Low Power node (LPN), giving the possibility of lowering the power consumption.
+
+While running the sample with the LPN configuration, the fourth :ref:`bt_mesh_onoff_cli_readme` instance will be omitted.
+Instead, the corresponding button will be used to temporarily enable Node Id advertiment on the LPN device.
+
+Running continuous proxy advertiment with Network Id consumes considerable power, and is therefore disabled in the configuration.
+Instead, the user can manually enable the Node Id advertiment for a period of 30 seconds by pressing **Button 4** on the device. This will give the user a short period where it is possible to connect directly to the LPN, and thus perform necessary configuration of the device.
```

Comment:
```suggestion
Running continuous proxy advertisement with Network ID consumes considerable power, and is therefore disabled in the configuration.
Instead, the user can manually enable the Node ID advertisement for a period of 30 seconds by pressing **Button 4** on the device.
This will give the user a short period of time to connect directly to the LPN, and thus perform necessary configuration of the device.
```